monitoring/heartbeats/expired-heartbeats.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Expired heartbeat messages
+summary: 
+reviewed: 2018-01-05
+component: Heartbeats
+versions: 'Heartbeats:*'
+---
+
+
+Heartbeat messages have a time to be received (TTBR) set based on the TTL value.  If ServiceControl does not consume the heartbeat messages before the TTBR expires then those messages may be discarded. Transports like MSMQ and ASB support DLQ and the expired heartbeat messages can be explicitly configured to be forwarded to the DLQ instead of being discarded. 
```

Comment:
Also duplicate

Reply:
Not sure what this means? Duplicate of what?